src/main/scala/fields/OptionalStringField.scala

```diff
@@ -1,5 +1,5 @@
 package fields
 
-case class OptionalStringField[Option[String]](f: Option[String]) extends AnyVal
+case class OptionalStringField[A](f: Option[String]) extends AnyVal
```

Comment:
Could maybe use `AnyVal[A =:= Option[String]]`. Not sure yet what that would do or if it would simplify to avoid writing: `OptionalStringField[OriginatorNoteId[Option[String]]]`?